scripts/utils/generate_strings.py

```diff
@@ -147,6 +171,20 @@ class I18nStrings final : public QQmlPropertyMap {
             """    __Last,
   };
 
+  static String getString(const QString& s) {
+  """
+        )
+
+        for i, key in enumerate(strings):
+            output.write(f"    {'if' if i == 0 else '} else if '} (s == \"{key}\") {{\n")
+            output.write(f"        return {key};\n")
+
+        output.write(
+            """    } else {
+          return Empty;
+        }
```

Comment:
```suggestion
      }
```

Alignment seems to be slightly off when looking at the .h file generated